commands/logs.py

```diff
@@ -0,0 +1,53 @@
+
+from errors import MalformedMoYaml
+from errors import UnknownEnvironment
+from subprocess import call
+import click
+import sys
+import yaml
+import io
+
+
+def command_logs(env):
+    """Executes the heroku logs command with the tail flag. The default
+    environment is staging but production can be tailed using the environment
+    flag
+    """
+    try:
+        # Open the mo.yml file
+        # TODO: Search recursively until we get to the root of the project
+        stream = io.open('mo.yml', 'r')
+        data = yaml.load(stream)
+        heroku_envs = data.get('heroku')
+
+        # If there is no heroku section of the mo.yml file, raise an error
+        if (heroku_envs is None):
+            raise MalformedMoYaml('No heroku apps defined for this project')
+
+        staging = heroku_envs.get('staging')
+        production = heroku_envs.get('production')
+
+        # Supported environments are staging and production
+        if (env != 'staging' and env != 'production'):
```

Comment:
This will be useful across  many commands, so we can eventually pull it out into a decorator for the command itself. Something like `@supported_envs('staging', 'production')`.